### PR TITLE
Air Tank Changes

### PR DESCRIPTION
**Pull request by FoxglovePurpur**
This is a pretty small one honestly and the changes can be summarized as such.
- Adds a unique sprite for cryo tanks. (They can't actually be accessed in game rn anyway though (yet))
- Changes the double nitrogen tank to use a unique sprite (for some reason it uses the same as double phoron...?)
- The old volumes were (sans oxy)
 - 6 for normal 10 for double
 - I've changed this to 12 for double, cuz it's double.
 - (Oxygen normal has a volume of 2, evidently) ((This is because phoron and nitrogen don't have extended tanks, so their "normal" is the extended version. Whereas oxygen gets emergency, emergency extended (the default for phoron/nitrogen), and double (which I did also change to 12).
> All number value changes and additions to the base non _vr file were just to make sure they match the vr file. I couldn't actually see if they're required to make things work or not. However, the sprite changes are required. Regardless, there was no issue with changes to it in my own tests.

### CODE REVIEW

File: code/game/objects/items/weapons/tanks/tank_types.dm [hunk 1]
```diff
@@ -153,14 +153,15 @@
 /obj/item/weapon/tank/emergency/oxygen/double
 	name = "double emergency oxygen tank"
 	icon_state = "emergency_double"
-	gauge_icon = "indicator_emergency_double"
-	volume = 10
+	gauge_icon = "indicator_double"
```
Comment (Razgriz1032) [inline code/game/objects/items/weapons/tanks/tank_types.dm:156]: CHOMP Edit here

File: code/game/objects/items/weapons/tanks/tank_types.dm [hunk 2]
```diff
@@ -153,14 +153,15 @@
...
+	volume = 12 		// CHOMP EDIT Double Expanded
 
 /obj/item/weapon/tank/stasis/oxygen // Stasis bags need to have initial pressure within safe bounds for human atmospheric pressure (NOT breath pressure)
 	name = "stasis oxygen tank"
 	desc = "Oxygen tank included in most stasis bag designs."
-	icon_state = "emergency_double"
-	gauge_icon = "indicator_emergency_double"
+	icon = 'icons/obj/tank_ch.dmi'			// CHOMP EDIT Adds a unique icon to the oxygen tank in stasis bags.
+	icon_state = "stasis_oxygen"
+	gauge_icon = "indicator_stasis"
```
Comment (Razgriz1032) [inline code/game/objects/items/weapons/tanks/tank_types.dm:164]: CHOMP Edits on these too I think

File: code/game/objects/items/weapons/tanks/tank_types.dm [hunk 3]
```diff
@@ -221,8 +222,9 @@
 /obj/item/weapon/tank/stasis/nitro_cryo // Synthmorph bags need to have initial pressure within safe bounds for human atmospheric pressure, but low temperature to stop unwanted degredation.
 	name = "stasis cryogenic nitrogen tank"
 	desc = "Cryogenic Nitrogen tank included in most synthmorph bag designs."
-	icon_state = "emergency_double_nitro"
-	gauge_icon = "indicator_emergency_double"
+	icon = 'icons/obj/tank_ch.dmi'		// CHOMP EDIT Adds a stasis sprite unique to this icon.
+	icon_state = "stasis_nitrogen"
+	gauge_icon = "indicator_stasis"
```
Comment (Razgriz1032) [inline code/game/objects/items/weapons/tanks/tank_types.dm:227]: CHOMP Adds here

**Comment by FoxglovePurpur** [comment]
Alright, sorry about that! I'll be sure to add a comment on each line with a change from now-on. TY for the corrections <3

**Pull request merged**